src/Unleash/Unleash.csproj

```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(OS)' != 'Windows_NT' ">
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
```

Comment:
Ideally we'd like to keep the amount of target frameworks down, we can get by on just netstandard2.0 and still support a major portion of the .NET versions out there which is why we had only that one